coursebuilder/app.yaml

```diff
@@ -1,5 +1,5 @@
 application: mycourse  # FIXME: Replace this with your application id
-version: 1
+version: 1 #tawakalt
```

Comment:
@Tawakalt as versions will change as each person pushes to the dev server, please revert this to the default.